samples/nrf5340/netboot/Kconfig

```diff
@@ -8,6 +8,19 @@ menu "nRF5340 Network Core Bootloader Sample Settings"
 
 config NETBOOT_MIN_PARTITION_SIZE
 	bool "Use minimimum partition size"
+
+config B0N_SIZE
+	hex
+	default 0x8580 if PM_PARTITION_SIZE_PROVISION=0x280
+	default 0x2580 if PM_PARTITION_SIZE_PROVISION=0x280 && NETBOOT_MIN_PARTITION_SIZE
+	default 0x8800
```

Comment:
not requesting changes atm, but need to take a deeper look into this.

We now have size coming from devicetree resulting in default Kconfig values, but those values are effectively being ignored because partition manager re-calculates.
PM sizes are then used in the build system, and apparently now being used as input to deciding default Kconfig size settings `B0N_SIZE`, but this size again is now used in the `pm.yml`.

This starts to look more and more messy, and hard to keep track of.